explainaboard/processors/conditional_generation.py

```diff
@@ -61,4 +60,14 @@ def __init__(self, metadata: dict, system_output_data: Iterable[dict]) -> None:
             metadata["metric_names"] = ["bleu"]
 
         super().__init__(metadata, system_output_data)
-        self._builder = SummExplainaboardBuilder(self._system_output_info, system_output_data)
\ No newline at end of file
+        self._builder = CondGenExplainaboardBuilder(self._system_output_info, system_output_data)
+
+
+@register_processor(TaskType.summarization)
+class SummarizationProcessor(ConditionalGenerationProcessor):
+    _task_type = TaskType.summarization
+
+
+@register_processor(TaskType.machine_translation)
+class MachineTranslationProcessor(ConditionalGenerationProcessor):
```

Comment:
All look good overall.
Notably, so far we assume that:
*  all [these features](https://github.com/neulab/ExplainaBoard/blob/17e8168c2688b24f2406f95b3d68b9e2e2d974b8/explainaboard/processors/conditional_generation.py#L14) are shared by all conditional text generation tasks. We can go with this first for immediate use, and refine them gradually (i.e., put more fundamental ones in the `ConditionalGenerationProcessor` class and task specifical one in `SummarizationProcessor` or `MachineTranslationProcessor`).
* We  regard `SummarizationProcessor`  as `text summarization` (since there will be other summarization tasks, such as video summarization.